CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.16.1]
+
+- Reverts v10.16.0 ([#15063](https://github.com/MetaMask/metamask-extension/pull/15063))
+- This reverts commit f09ab8889148c406551dea1643966e3331fde4aa, reversing ([#14912](https://github.com/MetaMask/metamask-extension/pull/14912))
+
```

Comment:
I assume this was a mistake? We don't need to list the revert here, it's not a change from a user's POV. 

```suggestion
```

Reply:
Hmm. I guess we can resolve this later when merging v10.16.0 back into this. We'll have to note then what's going on here.

Reply:
ah, yes, I have removed this